isic/core/serializers.py

```diff
@@ -1,12 +1,29 @@
 from rest_framework import serializers
+from rest_framework.fields import ListField
 
 from isic.core.models import Image
 from isic.core.models.image import RESTRICTED_SEARCH_FIELDS
 
 
+class StringArrayField(ListField):
```

Comment:
Could this use the same example from https://www.django-rest-framework.org/api-guide/fields/#listfield instead?